app/src/main/res/values/strings.xml

```diff
@@ -25,4 +25,5 @@
     <!-- Profile -->
     <string name="welcome_message">Hi %1$s!</string>
     <string name="sign_out">Sign Out</string>
+    <string name="default_error">Opps!!!, Network error!!!</string>
```

Comment:
typo en Oops! Also avoid extra punctuation characters and keep the translations correct as possible.
`Oops! Network error.` is better IMO.